packages/ui/src/helpers/authenticator/formFields/defaults.ts

```diff
@@ -105,11 +105,12 @@ const getConfirmSignUpFormFields = (state: AuthMachineState): FormFields => ({
 
 const getResetPasswordFormFields = (state: AuthMachineState): FormFields => {
   const primaryAlias = getPrimaryAlias(state);
+  const { label } = defaultFormFieldOptions[primaryAlias];
```

Comment:
This gets the label from here:
https://github.com/aws-amplify/amplify-ui/blob/932629520a2f855ae2cadbeb8ec741f5607276ef/packages/ui/src/helpers/authenticator/constants.ts#L89-L91